subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenVersionRangeResolveIntegrationTest.groovy

```diff
@@ -58,4 +58,43 @@ dependencies {
             }
         }
     }
+
+    @Issue("https://github.com/gradle/gradle/issues/1898")
+    def "can resolve parent pom with version range"() {
+        given:
+        settingsFile << "rootProject.name = 'test' "
+        buildFile << """
+repositories {
+    maven {
+        url "${mavenRepo.uri}"
+    }
+}
+
+configurations { compile }
+
+dependencies {
+    compile group: "org.test", name: "child", version: "1.0"
+}
+"""
+        and:
+        mavenRepo.module('org.test', 'child', '1.0').parent('org.test', 'parent', '[2.0,3.0)').publish()
+        mavenRepo.module('org.test', 'parent', '2.0').dependsOn('org.test', 'dep', '2.0').publishPom()
+        mavenRepo.module('org.test', 'parent', '2.1').dependsOn('org.test', 'dep', '2.1').publishPom()
```

Comment:
For completeness, I'd publish a version `3.0` here just to demonstrate that the range is being fully respected.

Reply:
Sure. I added `1.0` and `3.0`